edit/edit.js

```diff
@@ -0,0 +1,2068 @@
+        t('styleBeautifyIndentConditional') + '</label>' +
+      '</div>' +
+      '<div><button role="undo"></button></div>');
+
+    const undoButton = document.querySelector('#help-popup button[role="undo"]');
+    undoButton.textContent = t(scope.length == 1 ? 'undo' : 'undoGlobal');
+    undoButton.addEventListener('click', function() {
+      let undoable = false;
+      scope.forEach(function(cm) {
+        if (cm.beautifyChange && cm.beautifyChange[cm.changeGeneration()]) {
+          delete cm.beautifyChange[cm.changeGeneration()];
+          cm.undo();
+          cm.scrollIntoView(cm.getCursor());
+          undoable |= cm.beautifyChange[cm.changeGeneration()];
+        }
+      });
+      undoButton.disabled = !undoable;
+    });
+
+    scope.forEach(function(cm) {
+      setTimeout(function() {
+        const pos = options.translate_positions =
+          [].concat.apply([], cm.doc.sel.ranges.map(r =>
+            [Object.assign({}, r.anchor), Object.assign({}, r.head)]));
+        const text = cm.getValue();
+        const newText = exports.css_beautify(text, options);
+        if (newText != text) {
+          if (!cm.beautifyChange || !cm.beautifyChange[cm.changeGeneration()]) {
+            // clear the list if last change wasn't a css-beautify
+            cm.beautifyChange = {};
+          }
+          cm.setValue(newText);
+          const selections = [];
+          for (let i = 0; i < pos.length; i += 2) {
+            selections.push({anchor: pos[i], head: pos[i + 1]});
+          }
+          cm.setSelections(selections);
+          cm.beautifyChange[cm.changeGeneration()] = true;
+          undoButton.disabled = false;
+        }
+      }, 0);
+    });
+
+    document.querySelector('.beautify-options').onchange = ({target}) => {
+      const value = target.type == 'checkbox' ? target.checked : target.selectedIndex > 0;
+      prefs.set('editor.beautify', Object.assign(options, {[target.dataset.option]: value}));
+      if (target.parentNode.hasAttribute('newline')) {
+        target.parentNode.setAttribute('newline', value.toString());
+      }
+      doBeautify();
+    };
+
+    function optionHtml(label, optionName, indent) {
+      const value = options[optionName];
+      return '<div newline="' + value.toString() + '">' +
+        '<span' + (indent ? ' indent' : '') + '>' + label + '</span>' +
+        '<select data-option="' + optionName + '">' +
+          '<option' + (value ? '' : ' selected') + '>&nbsp;</option>' +
+          '<option' + (value ? ' selected' : '') + '>\\n</option>' +
+        '</select></div>';
+    }
+  }
+}
+
+document.addEventListener('DOMContentLoaded', init);
+
+function init() {
+  initCodeMirror();
+  const params = getParams();
+  if (!params.id) { // match should be 2 - one for the whole thing, one for the parentheses
+    // This is an add
+    tE('heading', 'addStyleTitle');
+    const section = {code: ''};
+    let i;
+    for (i in CssToProperty) {
```

Comment:
No need to separate the declaration. Also, use `const` because the value isn't changed in its block scope: `for (const i in CssProperty)`